rps.js

```diff
@@ -0,0 +1,37 @@
+///game :(
+
+function game(user) {
+
+
+    const pcNum = function () {
+        let pc = Math.floor(Math.random() * 3)
+        if (pc < 1) {
+            return "rock"
+
+        } else if (pc > 1) {
+            return "paper"
+        } else {
+            return "scissors"
+        }
+    }
+    
+    if (user === "rock " || user === "paper" || user === "scissers") {
+        if (user === pcNum()) {
```

Comment:
every time you call ```pcNum()``` you will get new value different from the previous call so we need to call ```pcNum()``` just one time and save the value into a variable. Then we comparing this variable with user input